.travis.yml

```diff
@@ -17,6 +17,7 @@ script:
   - npm run lint
   - npm run build
   - npm run test:ci
+  - npx commitlint-travis
```

Comment:
this line only lints the new commit messages?

Reply:
Correct - as intended